core/uvisor-config.h

```diff
@@ -27,7 +27,13 @@
  * link failure, so it can be used to track down a rough estimation of the
  * uVisor flash usage upper bound.
  */
-#define UVISOR_FLASH_LENGTH_MAX 0x9C00
+#if NDEBUG
+/* Release builds can be up to this big. */
```

Comment:
Good catch. I had a look into the release `*.linker` files and they are using the large debug limit. I'll have to set NDEBUG in the linker script generation step.